integration_tests/projects/013_structured_hooks/models/schema.yml

```diff
@@ -29,3 +29,17 @@ models:
                 - 3
               mapping:
                 key: value
+
+  - name: model_b
+    meta:
+      fal:
+        pre-hook:
+          - path: local_hook.py
+          - path: funny_hook.py
+            environment: funny
+        post-hook:
+          - path: check_imports.py
+            environment: not-funny
+            with:
+              import: pyjokes
```

Comment:
atl: what's the benefit of specifying imports here?

Reply:
Just to play with the combinations of arguments to hooks, and writing a hook that can actually be a test of itself.